src/components/forms/contactUs.tsx

```diff
@@ -99,6 +106,11 @@ const ContactUsForm: React.FC = () => {
                   value: option,
                   label: firstLetterUppercase(option),
                 }))}
+                defaultValue={
+                  Boolean(pageThatErrored)
```

Comment:
Why `Boolean(...)`? Using the `pageThatErrored` as is should do the trick right? Perhaps I'm wrong or there's some edge case.
```suggestion
                  pageThatErrored
```